CHANGELOG.md

```diff
@@ -897,6 +903,9 @@ The following two features, disabled by default, were mistakenly added to the re
 
 ## 8.21.0
 
+> ![Important]
+> This version introduced an issue for macOS apps that run without a sandbox (i.e. distributed outside the Mac App Store). We recommend upgrading to at least version 8.49.2.
```

Comment:
We should also update the GitHub release after merging and releasing this https://github.com/getsentry/sentry-cocoa/releases/tag/8.21.0.

Reply:
We should also consider adding a note to the hotfix release changes.
[Internal discussion](https://sentry.slack.com/archives/C08QHFV4W9E/p1745915959876679)

Reply:
Added a note to the unreleased section.